src/contracts/storage/tick.rs

```diff
@@ -382,7 +390,7 @@ mod tests {
             };
             let liquidity_delta = Liquidity::from_integer(3);
             let add = false;
-            tick.update_liquidity_change(liquidity_delta, add);
+            let _ = tick.update_liquidity_change(liquidity_delta, add);
```

Comment:
IMO, there should be `unwrap()` in these tests or `unwrap_err()` if the test is intended to throw an error, instead of skipping this result